readme: document metrics settings (#1449)
bottlerocket-os/bottlerocket: README.md

If you're running a Kubernetes variant, the no-proxy list will automatically include the Kubernetes API server endpoint and other commonly used Kubernetes DNS suffixes to facilitate intra-cluster networking.

#### Metrics settings

By default, Bottlerocket sends anonymous metrics when it boots, and once every six hours.
This can be disabled by setting `send-metrics` to false.
Here are the metrics settings:

> * `metrics.metrics-url`: the endpoint to which metrics will be sent.

Reviewer: super-duper-uber-minor nit: Can we capitalize the first word of the descriptions just to be consistent with the rest of the settings documentation?

> * `metrics.metrics-url`: the endpoint to which metrics will be sent.

Reviewer: Maybe we should also mention that this `metrics-url` is set by default as well. And it's set to send to a Bottlerocket-team-controlled endpoint (not sure how to word this).

> * `metrics.metrics-url`: The endpoint to which metrics will be sent. The default is `https://metrics.bottlerocket.aws/v1/metics`.

Reviewer 2: ```suggestion
* `metrics.metrics-url`: The endpoint to which metrics will be sent. The default is `https://metrics.bottlerocket.aws/v1/metrics`.
```

* `metrics.send-metrics`: whether or not Bottlerocket will send anonymous metrics.

> * `metrics.service-checks`: the systemd services that will be checked to determine whether a host is healthy or not.

Reviewer: ```suggestion
* `metrics.service-checks`: A list of systemd services that will be checked to determine whether a host is healthy or not.
```

> * `metrics.service-checks`: A list of systemd services that will be checked to determine whether a host is healthy or not.

Reviewer 3: We typically prepend `settings.` to these values.

> * `metrics.service-checks`: A list of systemd services that will be checked to determine whether a host is healthy or not.

Reviewer 2: ```suggestion
* `metrics.service-checks`: A list of systemd services that will be checked to determine whether a host is healthy.
```
nit: "whether" usually implies "or not".  (same in send-metrics, if you don't hate this suggestion.)
